pulpcore/pulpcore/app/response.py

```diff
@@ -31,7 +31,8 @@ def __init__(self, task_results, request):
         """
         tasks = []
         for result in task_results:
-            task = {"_href": reverse('tasks-detail', args=[result.task_id], request=request),
+            task = {"_href": reverse('tasks-core-updates-detail', args=[result.task_id],
+                                     request=request),
```

Comment:
TODO: Can this be generalized? 

something like:

    task = CoreTask.objects.get(id=result.task_id).cast()
    
Except I don't think the model object knows about its serializer. Could we find some way to use a serializer to have DetailRelatedField(many=True) that would cast() to render with the detail serializer? Would replace "OperationPostponedResponse"